examples/text_classification/iterable_train.py

```diff
@@ -263,21 +275,30 @@ def count(data_path):
     logging.info("Counting testing lines and labels")
     num_labels, test_num_lines = count(test_data_path)
 
+    # Split training dataset into train and valid
+    train_len = int(train_num_lines * split_ratio)
```

Comment:
I assume you tests this with split_ratio 1?

Reply:
Actually not here. Let me do it and see how it works.